.drone.yml

```diff
@@ -172,6 +172,27 @@ steps:
           - feature/*
       event: [push, pull_request]
 
+# Trivy Security Scannner
+
+  - name: scan-image
+    pull: always
+    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
+    resources:
+      limits:
+        cpu: 1000
+        memory: 1024Mi
+    environment:
+      IMAGE_NAME: ima:${DRONE_COMMIT_SHA}
+      TOLERATE: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
```

Comment:
I don't think we should tolerate high and critical  and should we add a WHITELIST_FILE for privy exceptions?